spec/line/bot/v2/misc_spec.rb

```diff
@@ -1581,7 +1580,7 @@
     end
 
     context 'with invalid enum values' do
-      it "doesn't raise any error" do
+      it "doesn't raise any error", rbs_test: :skip do
```

Comment:
rbs test fails when unknown const value(enum) is came, because unknown value is not in definition. Thus we skip this test in rbs test